app/controllers/users_controller.rb

```diff
@@ -7,6 +7,8 @@ def edit
 
   def update
     if @user.update_attributes(user_params)
+      sign_in :user, @user, bypass: true
```

Comment:
@PaulStorberg Only sign in user if the user is kicked out because of a password update.